job-dsl-plugin/src/main/groovy/javaposse/jobdsl/plugin/ExecuteDslScripts.java

```diff
@@ -302,7 +313,7 @@ public void perform(@Nonnull Run<?, ?> run, @Nonnull FilePath workspace, @Nonnul
                     dslScriptLoader = new JenkinsDslScriptLoader(jobManagement);
                 }
 
-                GeneratedItems generatedItems = dslScriptLoader.runScripts(scriptRequests);
+                GeneratedItems generatedItems = dslScriptLoader.runScripts(scriptRequests, additionalParameters);
```

Comment:
This PR can be much simpler by just putting the additional parameters into the existing environment map, see https://github.com/jenkinsci/job-dsl-plugin/blob/job-dsl-1.63/job-dsl-plugin/src/main/groovy/javaposse/jobdsl/plugin/ExecuteDslScripts.java#L276-L279.